src/components/cylc/Graph.vue

```diff
@@ -0,0 +1,1644 @@
+          }
+          if (has(node, 'jobs') && !isEmpty(node.jobs)) {
+            jobsArray = this.getJobs(node.jobs)
+            nodeObj.data.jobsarray = jobsArray
+            const joblength = node.jobs.length
+            nodeObj.data.jobs = String(node.jobs.length)
+            nodeObj.data.submittedtime = jobsArray[0].submittedTime
+            nodeObj.data.starttime = jobsArray[0].startedTime
+            nodeObj.data.finishtime = jobsArray[joblength - 1].finishedTime
+          }
+          nodesArray.push(nodeObj)
+        })
+        // console.debug('NODES ::: ', JSON.stringify(nodesArray))
+        return nodesArray
+      } catch (error) {
+        console.error('getNodes error: ', error)
+      }
+    },
+
+    getEdges (edges) {
+      try {
+        const edgesArray = []
+        let edgeObj = {}
+        each(edges, (edge, key) => {
+          edgeObj = {
+            data: {
+              id: '',
+              source: '',
+              target: '',
+              label: '',
+              owner: '',
+              name: ''
+            },
+            position: {
+
+            },
+            group: 'edges',
+            removed: false,
+            selected: false,
+            selectable: true,
+            locked: false,
+            grabbable: true,
+            classes: ''
+          }
+          has(edge, 'id') && !isEmpty(edge.id) ? edgeObj.data.id = edge.id : console.debug('workflowUpdated - edge id is empty')
+          has(edge, 'source') && !isEmpty(edge.source) ? edgeObj.data.source = edge.source : edgeObj.source = undefined
+          has(edge, 'target') && !isEmpty(edge.target) ? edgeObj.data.target = edge.target : edge.target = undefined
+          if (!isEmpty(this.workflowOwner)) {
+            edgeObj.data.owner = this.workflowOwner
+          }
+          if (!isEmpty(this.workflowName)) {
+            edgeObj.data.name = this.workflowName
+          }
+          has(edge, 'label') && !isEmpty(edge.label) ? edgeObj.data.label = edge.label : edgeObj.label = ''
+          edgeObj.data.source !== undefined || edgeObj.data.target !== undefined ? edgesArray.push(edgeObj)
+            : console.debug('skipping adding edge with empty source or target')
+        })
+        // console.debug('EDGES ::: ', JSON.stringify(edgesArray))
+        return edgesArray
+      } catch (error) {
+        console.error('getEdges error: ', error)
+      }
+    },
+
+    getJobs (jobs) {
+      const jobsArray = []
+      each(jobs, (job, key) => {
+        jobsArray.push(job)
+      })
+      return jobsArray
+    },
+
+    changeLayout (value) {
+      try {
+        if (this.freeze) {
+          return
+        }
+        if (!isUndefined(value) && !isEmpty(value)) {
+          this.layoutName = value
+        }
+      } catch (error) {
+        console.error('changeLayout error: ', error)
+      }
+    },
+
+    async handleMounted () {
+      try {
+        this.status = 'success'
```

Comment:
Was there some other code here before, that could throw errors? Right now I think this `handleMounted` could be removed? The `catch` above is probably never executed?

Reply:
yep this is a left over